packages/ownable/src/lib.rs

```diff
@@ -79,7 +256,8 @@ pub enum OwnershipError {
 }
 
 /// Storage constant for the contract's ownership
-const OWNERSHIP: Item<Ownership<Addr>> = Item::new("ownership");
+pub const OWNERSHIP_KEY: &str = "ownership";
+pub const OWNERSHIP: OwnershipStore = OwnershipStore::new(OWNERSHIP_KEY);
```

Comment:
Doesnt hurt for making it public. Makes it easier e.g. for migration. In ICS721 I had to use a workaround, since this wasnt public.